### DISCUSSION TITLE
GitHub Mobile in China

### DISCUSSION THREAD

**Discussion by xiaohh-me** [community/community#63228] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

We hope to open GitHub Mobile in China
![IMG_8EF53D639144-1](https://github.com/community/community/assets/60495157/f0ceba2c-e6de-4eea-bea1-d00ab70117c0)

**Comment by eliperkins** [comment, marked as answer]
Hey @xiaohh-me, thanks for this feedback.

Unfortunately, GitHub for iOS is not available in China. I've added this feedback to our internal tracking repo, so I'll be sure to reach out if the situation changes!
